mindmeld/text_preparation/text_preparation_pipeline.py

```diff
@@ -627,3 +663,20 @@ def _construct_pipeline_component(  # pylint: disable=W0640
             raise TypeError(
                 f"{component} must be of type String or {expected_component_class.__name__}."
             )
+
+
+class TPPJSONEncoder(json.JSONEncoder):
```

Comment:
I think its better to state the full form here: `TextPreparationPipelineJSONEncoder`

Reply:
Done